react/components/EditorContainer/Sidebar/ComponentList/SortableList/SortableListItem/index.tsx

```diff
@@ -51,13 +70,13 @@ class SortableListItem extends Component<Props, State> {
           onMouseEnter={onMouseEnter}
           onMouseLeave={onMouseLeave}
         >
-          {shouldRenderDragHandle && component.isSortable && (
+          {shouldRenderOptions && component.isSortable && (
```

Comment:
Could use `shouldRenderActionMenu` here

Reply:
Yes, but it would look a bit weird since it controls whether or not the `DragHandle` should be rendered, and not `ActionMenu`. However, I do agree that the rendering conditions could be more "semantic".